spec/honeycomb/integrations/rack_spec.rb

```diff
@@ -1,31 +1,41 @@
 # frozen_string_literal: true
 
 if defined?(Honeycomb::Rack)
+  # Rack::Session was moved to a separate gem in Rack v3.0
+  if Gem::Version.new(::Rack.release) >= Gem::Version.new("3.0")
+    require "rack/session"
+  end
   require "rack/test"
-  require "rack/lobster"
   require "warden"
 
+  class AnApp
+    def call(_env)
```

Comment:
Starting the unused variable with an underscore will appease the Lint/UnusedMethodArgument Rubocop cop.

Reply:
TIL